index.js

```diff
@@ -1,9 +1,38 @@
 var scrape = require("./scrape");
 var express = require("express");
 var pkgInfo = require("./package.json");
+var html2md = require("html-md");
+var markdown = require("markdown");
 
 var app = express();
 
+/**
+ * Casts a qs string arg into an actual boolean.
+ * @param  {String} arg The query string arg.
+ * @return {Boolean}
+ */
+function boolArg(queryParam) {
+  if (!queryParam) return false;
+  return ["1", "on", "true", "yes", "y"].indexOf(queryParam.toLowerCase()) !== -1;
+}
+
+/**
+ * Takes a result object and replace native html contents with a safer sanitized
+ * version.
+ * @param  {Object} resultObject
+ * @return {Object}
+ */
+function sanitizeResult(resultObject) {
```

Comment:
I debated that, though as you can see with my update following your previous comment I need to update the resultObject for the modified size as well.